src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -263,29 +255,29 @@ private void SidebarControl_SidebarItemInvoked(object sender, SidebarItemInvoked
 			switch ((invokedItemContainer.DataContext as INavigationControlItem)?.ItemType)
 			{
 				case NavigationControlItemType.Location:
-					{
```

Comment:
Were these supposed to be changed?

Reply:
Sorry. It seems to be the result of code formatting. I will revert it later.